msteams-platform/bots/how-to/Teams conversational AI/assistants-api-quick-start.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Assistants API quick start guide
+author: surbhigupta
+description: In this module, learn how to quickly try the Assistants API.
+ms.topic: conceptual
+ms.localizationpriority: high
+zone_pivot_groups: assistant-ai-library-quick-start
+ms.author: v-bvishnu
+ms.date: 05/20/2024
+---
+
+# Assistants API quick start guide
+
+Get started with the OpenAI Assistants API using the MathBot sample. This guide uses the OpenAI Code Interpreter tool to help you create an assistant that specializes in mathematics. The bot uses the gpt-3.5-turbo model to chat with Microsoft Teams users and respond in a polite and respectful manner, staying within the scope of the conversation.
```

Comment:
Get started using OpenAI or Azure OpenAI Assistants API with Teams AI library in MathBot sample.